packages/components/addon/components/hds/side-nav/base.hbs

```diff
@@ -9,10 +9,10 @@
     <div class="hds-side-nav__wrapper-header">
       {{yield to="header"}}
     </div>
-    <div class="hds-side-nav__wrapper-body">
+    <div class="hds-side-nav__wrapper-body" inert={{@isMinimized}}>
       {{yield to="body"}}
     </div>
-    <div class="hds-side-nav__wrapper-footer">
+    <div class="hds-side-nav__wrapper-footer" inert={{@isMinimized}}>
```

Comment:
Let's imagine that consumers want to add something that persists in the UI even when the sidenav is collapsed (do you remember we have a lot of "extra" content slots in the sidenav, you can see them in the [demo app](https://hds-showcase-git-alex-ju-local-demo-app-hashicorp.vercel.app/demo-app) too). 
From [this quick test](https://codepen.io/didoo/pen/NWJwvNb) they would not be able to assign an `inert="false"` to their custom content to bypass the parent's `inert="true"`.

Thoughts of how we could bypass this limitation? 🤔

Reply:
it's not something to be bypassed, it's a constraint that consumers have to understand and accept as well.

Reply:
@MelSumner the point is that they may have _extra_ content that is correct semantically and functionally, is fully accessible, and for [reasons] they want to leave it available even when the sidenav is collapsed (think of an extra button, for example to quickly open a CLI). if we make the entire `body` and `footer` inert, this would become inert too. my question was about bypassing the limitation in our solution. In theory if it was a CSS property we could tell them to add a special class to it, like we already do with the `hds-side-nav-hide-when-minimized` class ([reference](https://helios.hashicorp.design/components/side-nav?tab=code#content-transitions-between-states)) but since this is an HTML attribute I can't think of other ways to do it.

Reply:
The only element that can escape inertness is the modal dialog.

Alternatively to this PR, we can offer guidance and ask the consumer to manage the inert state. Of course, it comes with tradeoffs (if they mismanage it, it becomes unusable or non-conformant).

For portal content

```hbs
<Hds::SideNav>
  <:body as |Body|>
    <Hds::SideNav::Portal::Target inert={{Body.isMinimized}} @targetName="sidenav-portal" />
  </:body>
</Hds::SideNav>
```

For list content

```hbs
<Hds::SideNav>
  <:body as |Body|>
    <Hds::SideNav::List aria-label="Services" inert={{Body.isMinimized}} as |SNL|>
      <SNL.Title>Services</SNL.Title>
      <SNL.Link @text="Boundary" @icon="boundary" @href="#" />
      <SNL.Link @text="Consul" @icon="consul" @href="#" />
    </Hds::SideNav::List>
  </:body>
</Hds::SideNav>
```

Reply:
What if apply `inert` via JS, like `[DOM Element].setAttribute('inert', true)` (and we can point to specific selectors of things we know we want to make inert, and when, in the `index.js` code?

Reply:
The problem with us setting inert only on certain elements is that the consumer may not be aware of it and will have no way to opt-in with other elements to become inert/non-inert based on the collapsed/expanded state of the nav. We only need one element (that is not visible when the SideNav is collapsed) to miss the inert attribute and the implementation is non-compliant.

Reply:
I'll give it some more thought, but I don't think the approach in this PR allows for exceptions

Reply:
Updated this PR (and the demo app) to make elements with `.hds-side-nav-hide-when-minimized` inert when the SideNav is collapsed. This way we can have parity between hidden elements and the inert attribute, regardless of whether the class is set within the component, or by the consumer.

Reply:
Yes!!!! I was sure you would have found the "idiomatic" solution in the end! Great job 👏 👏 👏